src/App.js

```diff
@@ -39,59 +39,57 @@ export default function App() {
             <Router>
               <NavigationBar />
               <Container className="pawhub">
-                <Switch>
-                  <Route path="/animal/:id">
-                    <PetLoading>
-                      <PetInfo />
-                    </PetLoading>
-                  </Route>
-                  <Route path="/pets/:type">
-                    <PetLoading>
-                      <PetType />
-                    </PetLoading>
-                  </Route>
-                  <Route path="/pets">
-                    <Pets />
-                  </Route>
-                  <Route path="/about">
-                    <About />
-                  </Route>
-                  <Route path="/resources">
-                    <Resources />
-                  </Route>
-                  <Route path="/donate">
-                    <Donate />
-                  </Route>
-                  <Route path="/stories">
-                    <Stories />
-                  </Route>
-                  <Route path="/register" component={Register} />
-                  <Route exact path="/login" component={SLogin} />
+                <Routes>
                   <Route
-                    exact
-                    path="/forgot-password"
-                    component={ForgotPassword}
+                    path="animal/:id"
+                    element={
+                      <PetLoading>
+                        {" "}
+                        <PetInfo />{" "}
+                      </PetLoading>
+                    }
                   />
-                  <PrivateRoute
-                    exact
-                    path="/reset-password"
-                    component={ResetPassword}
+                  <Route
+                    path="pets/:type"
+                    element={
+                      <PetLoading>
+                        {" "}
+                        <PetType />{" "}
+                      </PetLoading>
+                    }
                   />
-                  <Route exact path="/profile">
-                    <SupaLoading>
-                      <Profile />
-                    </SupaLoading>
-                  </Route>
-                  <Route path="/" exact>
-                    <PetLoading>
-                      <Home />
-                    </PetLoading>
+                  <Route path="pets" element={<Pets />} />
+                  <Route path="about" element={<About />} />
+                  <Route path="resources" element={<Resources />} />
+                  <Route path="donate" element={<Donate />} />
+                  <Route path="stories" element={<Stories />} />
+                  <Route path="register" element={<Register />} />
+                  <Route path="login" element={<SLogin />} />
+                  <Route path="forgot-password" element={<ForgotPassword />} />
+                  <Route element={<PrivateRoute />}>
+                    <Route path="reset-password" element={<ResetPassword />} />
```

Comment:
Looks like we the changes, I think Private Route needs some changes.

we would have `<Route path="reset-password" element={<PrivateRoute ><ResetPassword/></PrivateRoute>}>`